sdk/eventhub/event-hubs/CHANGELOG.md

```diff
@@ -187,36 +187,32 @@ Construction of both objects is the same as it was for the previous client.
 
 Version 5.0.0-preview.1 is a preview of our efforts to create a client library that is user friendly and
 idiomatic to the Javascript ecosystem. The reasons for most of the changes in this update can be found in the
-[Azure SDK Design Guidelines for TypeScript](https://azuresdkspecs.z5.web.core.windows.net/TypeScriptSpec.html).
+[Azure SDK Design Guidelines for TypeScript](https://azure.github.io/azure-sdk/typescript_introduction.html).
 For more information, please visit https://aka.ms/azure-sdk-preview1-js
```

Comment:
This aka doesn't follow the pattern.

Reply:
>  https://aka.ms/azure-sdk-preview1-js

Reply:
This isn't an aka. I thought that the TypeScript guidelines didn't need an aka.ms.

Reply:
This: For more information, please visit https://aka.ms/azure-sdk-preview1-js

Reply:
I have this as a question on the PR description:

> To replace the TypeScript guidelines from their older link, https://azuresdkspecs.z5.web.core.windows.net/TypeScriptSpec.html, to their actual link https://azure.github.io/azure-sdk/typescript_introduction.html. Is this change correct? Should I make this another aka.ms link?

Reply:
@jongio why is it preview1 if it's general guidelines?

Reply:
IMO, we don't need that whole line... "For more information, please visit https://aka.ms/azure-sdk-preview1-js"

Reply:
@jongio I just saw this:

> This: For more information, please visit https://aka.ms/azure-sdk-preview1-js

I think GitHub wasn't loading on me earlier! I understand your point now

Reply:
Ok! Removing that line. I'll remove similar lines everywhere.

Reply:
What about changing these https://aka.ms/azure-sdk-preview1-js to something like https://aka.ms/azsdk/releases/July2019Preview ? Otherwise some changelogs are left with empty entries.

Reply:
I'll change it to that for now.

Reply:
I've done this preemptively. Hoping this help us save some time.

Reply:
Sounds good, we now track releases per month not as ordered sequence